src/content/docs/en/guides/routing.mdx

```diff
@@ -296,6 +296,32 @@ if (!isLoggedIn(cookie)) {
 </html>
 ```
 
+## Rewrites
+
+<Since v="4.13.0" />
+
+A rewrite allows you serve a different route without redirecting the browser to a different page. The browser will show the same address in the URL bar, but will display the content of the URL provided to `Astro.rewrite()`.
+
+This can be useful for showing the same content at multiple paths (e.g. `/products/shoes/men/` and `/products/men/shoes`/).
+
+For content that has permanently moved, or to direct your user to a different page (e.g. a user dashboard after logging in), use a [redirect](#redirects) instead.
+
+A rewrite is really useful in an i18n context, for example. Let's suppose your website serves localised version of Spanish e.g. `/es-es/articles/introduction` (Spain) and `/es-mx/articles/introduction` (Mexico). However, your website also serves pages for `es-CU` (Cuba), but you're missing the localised content for `/es-cu/articles/introduction`. 
+
+However, you don't to serve that page as a 404, and you don't want to serve a redirect for SEO purpose. That's where the rewrites comes into play. You can decide to **fallback** to the Mexican localised page using the new `Astro.redirect` method:
+
+```astro title="src/pages/es-cu/articles/introduction.astro"
+---
+return Astro.rewrite("/es-mx/articles/introduction")
+---
+```
+
+When you'll navigate the URL `/es-cu/articles/introduction`, Astro will render the route `src/pages/es-mx/articles/introduction.astro`.
+
+The function `Astro.redirect()` will trigger a new complete rendering phase, which means that, if you have middleware, it will be executed again for the new route/request. 
```

Comment:
This feels like maybe it should go somewhere before the example, as it's more general about how it works. It also feels like it might get lost or grouped in with the example, when this is a standalone fact. I'll try moving it up and see what we think! 

```suggestion
```